content/python/concepts/sets/terms/update/update.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.update()'
+Description: 'Updates the current set, by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+The `.update()` method updates the current [set](https://www.codecademy.com/resources/docs/python/sets), by adding items from another iterable(set, list, tuple or dictionary), removing any duplicates.
+
+## Syntax
+
+```pseudo
+set.update(iterable)
+```
+
+Or, the alternative syntax is:
+
+```pseudo
+set |= iterable
+```
+
+- The `.update()` method can be called directly against `set` with multiple `iterable` passed as arguments.
+- `iterable` can be a `set`, `tuple`, `list` or `dictionary`.
+- Optionally,the `|=` can be used as a shortcut of `update()` and the `|` [operator](https://www.codecademy.com/resources/docs/python/operators) can be used between `iterable1`, `iterable2` where it will return the same result as the `.update()` method.
```

Comment:
This line can be removed.

Reply:
which line exactly ? or all of them?

Reply:
I removed all of them and added a note as you mentioned in the other change